WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderTagHeaderView.kt

```diff
@@ -28,32 +26,21 @@ class ReaderTagHeaderView @JvmOverloads constructor(
     @Inject
     lateinit var uiHelpers: UiHelpers
 
-    @Inject
-    lateinit var readerImprovementsFeatureConfig: ReaderImprovementsFeatureConfig
-
     @Inject
     lateinit var localeProvider: LocaleProvider
 
     private var onFollowBtnClicked: (() -> Unit)? = null
 
     init {
         (context.applicationContext as WordPress).component().inject(this)
-        binding = if (readerImprovementsFeatureConfig.isEnabled()) {
-            val readerTagHeaderViewNewBinding =
-                ReaderTagHeaderViewNewBinding.inflate(LayoutInflater.from(context), this, true)
+        val readerTagHeaderViewBinding =
+            ReaderTagHeaderViewBinding.inflate(LayoutInflater.from(context), this, true)
+        binding =
             ReaderTagBinding.ImprovementsEnabled(
```

Comment:
Now that we have just one implementation, what do you think about making `ReaderTagBinding` a concrete class and moving the implementation from this subclass to there?

Also, delete the now unused `ImprovementsDisabled` class (which will be necessary anyway if you implement the suggestion above).